core/lib/dal/src/blocks_dal.rs

```diff
@@ -1574,8 +1581,7 @@ impl BlocksDal<'_, '_> {
         default_aa_hash: H256,
         protocol_version_id: ProtocolVersionId,
     ) -> anyhow::Result<Vec<L1BatchWithMetadata>> {
-        let raw_batches = sqlx::query_as!(
-            StorageL1Batch,
+        let raw_batches = sqlx::query_as::<_, StorageL1Batch>(
```

Comment:
Hmm, why are you changing macro to function?

Reply:
The reason for this is the `query_as!` macro [needs the struct field names to be the exact same as the ones in postgres](https://docs.rs/sqlx/latest/sqlx/macro.query_as.html), but with the new columns mapped to the old struct fields, so the names do not match anymore. The `query_as` function allows for this